### PR TITLE
Feature/development

### PR DESCRIPTION
**Pull request by RokoVarano**
This is our work so far. We closed the previous Pull request to fix some Lint errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+
+- GitHub: @RokoVarano
+- Twitter: @RodrigoIbacet11
+- LinkedIn: https://www.linkedin.com/in/rodrigo-ibaceta-a8657611a/
+
+Manuel Aldaraca
+
+- Github: @vicmaburrito
+- Twitter: @ManuelAldaraca 
+- LinkedIn: https://www.linkedin.com/in/manuel-aldaraca
```
Comment (descholar-ceo) [inline README.md]: These social networks, should be written as links

File: index.html [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+<!--                    </a>-->
+<!--                    <a href="https://www.nytimes.com/article/top-book-lists.html?action=click&algo=top_conversion&block=editors_picks_recirc&fellback=true&imp_id=877282866&impression_id=27848652-2f39-11eb-998e-255129888bff&index=2&pgtype=Article&region=ccolumn&req_id=160481777&surface=home-featured">-->
+<!--                        <img src="assets/article3.jpg">-->
+<!--                        <h5> The 10 best books Through time </h5>-->
+<!--                    </a>-->
+<!--                </div>-->
+<!--            </aside>-->
+<!--            <p>My family and I visited The Epcot center this summer. The weather was nice, and Epcot was amazing! I had a great summer together with my family!</p>-->
+<!--            <p>My family and I visited The Epcot center this summer. The weather was nice, and Epcot was amazing! I had a great summer together with my family!</p>-->
+<!--    </div>-->
```
Comment (descholar-ceo) [inline index.html]: Leaving commented codes in your project is highly discouraged, please remove them

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @RokoVarano and @vicmaburrito  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone almost looks like the original page and it is really nice, congratulations on the good work you have done so far.
### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 

- [x] The links to your social networks should be clickable in your readme file
- [x] Your readme file is missing the screenshot of your project, and live demo link
- [x] Your readme file should follow the [microverse's readme template](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md)
- [ ] You should fix the linters errors before re-submitting for re-review :point_right: I looked at all of the lint errors you have, they are fixable, all you need is to sit and work on them until they are fixed
- [x] Remove `.idea` folder from your repository, that is not related to this project but your IDE, so it's unprofessional to let unrelated files/folders in your repository :point_right: you can do that by following the below steps
```bash
git rm -rf .idea
git commit -am "Remove idea from the repo"
git push
```
- [x] Your project is missing a big part, from `Suggested newsletters for you` to footer, that part should be worked on as well
- [x] The `LOG IN` button in navbar has a stray less than `<` symbol, remove it.
- [x] You used this `#titlepicks` which is an ID as a CSS selector, that is a bad practice, please use classes.
- [x] CSS codes should be DRY (Don't Repeat Yourself), currently there are some properties which are being repeated several times in your CSS codes, example: `display: flex;` is repeated 10 times, this is bad and makes your code difficult to maintain. 

#### Advice: 
1. Whenever you see that you need to apply a specific property on different elements, create a specific class which hold that property, and then apply that class to whichever element you need,
2. Or you can just group those elements and apply that property at once,

- [x] You have used inline styles from your html codes, that is a bad practice, please move all of the stylings in your CSS codes

- [ ] Using `flex` on this project is very good and encouraged since it is the main topic of it, but also we need you to learn and master how to use `float` and `grid`, currently there is no `grid` and no `float` in your CSS codes, try to use some
- [x] You have commented codes in your HTML file, please remove them
- [ ] There is no `Editor's pick` aside section on the rightmost side of the page
- [ ] You should use  [aside tag](https://www.w3schools.com/tags/tag_aside.asp) for Editor's picks section on the righmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))
- [x] The social media icons don't seem like the ones on the original page

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-11-26 01-18-28 | Screenshot from 2020-11-26 01-18-44 |


:point_right: Could you see the difference :question: :question: :question: 

- [ ] Work again on this Pull request description, actually you should talk about the work you have done on it in summary,

- [ ] Implement all changes I suggested in inline comments in your codes

# [Optional changes] :yellow_circle: 
- [ ] Your commit messages should be consistently in imperative mode or present tense, _i.e.:_ you should have written `Add more images of the article`  instead of writing `added more images of the article`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

=======
If anything seems to be unclear to you, comment on this Pull Request and mention me as [@descholar-ceo](https://github.com/descholar-ceo)
 | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Comment by RokoVarano** [comment]
- Closing this pull request so linterns can be used again. Thanks for the advice!

**Comment by RokoVarano** [comment]
Finished adding content to the project. The CSS Lint problems are due to it taking in the fontAwesome CSS files. We have added the directory into .gitignore, but it doesn't work. We are currently working to solve the issue.

- The Editor's Notes section to the right of the article has been added
- The footer of the article has been added
- The recommended newsletter section at the bottom of the article has been added
- The link menu at the very bottom of the website has been added.

**Comment by RokoVarano** [comment]
Solved the FontAwesome issue

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
Hello team :wave: 
You have done a nice job on this project. 
There are still some changes to be made.
- [x] In the "More in Space and Astronomy" section, you should add more margin between the elements as it is in the original. Also, in the "Most popular" section try to make the font-weight be as close to those used in the original website in that section.
- [x] Also, do not forget to fix the linter issues you have.

Once you have done all this, do go ahead and request another review.

slack(@ebeagusamuel)

**Comment by RokoVarano** [comment]
- Suggested changes have been made.
- Lint reports no problems

**Review by Srjnnnn — APPROVED** [review]
## Status: Approved :partying_face: :tada: 
Hello team! Thanks for making the changes requested from the previous TSE. Your page looks amazing. :muscle: 
You have made all the mandatory requirements.
You can merge this branch and move on to the next project. :+1: 
Eyüp Sercan UYGUR

**Pull request merged**